docs/manuals/core/config/scenario-config.md

```diff
@@ -64,3 +64,22 @@ associated to a Cycle corresponding to their creation date. See documentation on
 [Scenario and cycle management](../entities/scenario-cycle-mgt.md).
 
 [:material-arrow-right: The next section introduces the global configuration](global-config.md).
+
+# Configuring scenario comparators
+
+Let us imagine a common situation where the pipeline configuration `pipeline_config` has been created with `datanode_config` as one of the data node configuration for this the pipeline configuration.
+The function `compare_function` is also defined as followed:
+
+```python
+from taipy import Config
+
+def compare_function(*data_node_results):
+  comparison_result = data_node_results[0]
+  for data_node in data_node_results[1:]:
+    comparison_result -= data_node
+  return comparison_result
```

Comment:
What about computing all possible comparisons? Something like:
```
# Calling compare_function(10, 13, 17, 9) returns the following dict
# {
# 0: {0: 0, 1: 3, 2: 7, 3: -1}, 
# 1: {0: -3, 1: 0, 2: 4, 3: -4}, 
# 2: {0: -7, 1: -4, 2: 0, 3: -8}, 
# 3: {0: 1, 1: 4, 2: 8, 3: 0}}
def compare_function(*data_node_results):
    comparison_result= {}
    current_result_index = 0
    for current_result in data_node_results:
        comparison_result[current_result_index]={}
        next_result_index = 0
        for next_result in data_node_results:
            print(f"comparing result {current_result_index} with result {next_result_index}")
            comparison_result[current_result_index][next_result_index] = next_result - current_result
            next_result_index += 1
        current_result_index += 1
    return comparison_result
```

May be it is too much. What do you think?

Reply:
@jrobinAV 
The datanode_config is very generic, so I went with something generic as well while keeping it simple so that it's easy to understand. But we can definitely make the compare function more complicated! Let me know what you think :D should we make it more complicated or the current version is good enough? I'll also put an example input and output at the beginning of the function like you did to make it extra clear 😄

Reply:
Okay I'll take it back, it looks weird when saying "Calling the compare_function(10, 13, 17, 9) will return -29" 🤣